config.toml

```diff
@@ -24,9 +24,11 @@ github-repos = ["brson/hello-rs"]
 # These sections allows to customize how crater treats specific crates/repos
 #
 # The available options for each crate/repo are:
-#  - skip       (bool): ignore this crate/repo
-#  - skip-tests (bool): don't run tests in this crate/repo
-#  - quiet      (bool): don't kill after two minutes without output
+#  - skip            (bool): ignore this crate/repo
+#  - skip-tests      (bool): don't run tests in this crate/repo
+#  - quiet           (bool): don't kill after two minutes without output
+#  - update-lockfile (bool): update the lockfile even if the crate has one
+#  - broken          (bool): skip the crate only if there is an crater error during a stage
```

Comment:
Can we rephrase this a bit? Maybe "Treat a crater error on this crate as a build failure (typically the crate is broken in an unusual way and we want to indicate the failure is 'permissible')" - you might need to word wrap.

Or something shorter if it seems appropriate. It was just initially confusing to puzzle out what we're trying to achieve by transforming an 'Error' into a 'BuildFail' until I remembered your (reasonable) position on making unexpected errors loud.